src/storage/mvcc/mod.rs

```diff
@@ -299,15 +309,25 @@ pub mod tests {
         for_update_ts: u64,
         is_pessimistic_lock: bool,
     ) {
-        must_prewrite_put_impl(
-            engine,
-            key,
-            value,
-            pk,
-            ts,
-            for_update_ts,
-            is_pessimistic_lock,
-        );
+        let mut options = Options::default();
+        options.for_update_ts = for_update_ts;
+        must_prewrite_put_impl(engine, key, value, pk, ts, is_pessimistic_lock, options);
+    }
+
+    pub fn must_prewrite_put_for_large_txn<E: Engine>(
+        engine: &E,
+        key: &[u8],
+        value: &[u8],
+        pk: &[u8],
+        ts: u64,
+        ttl: u64,
+        for_update_ts: u64,
+    ) {
+        let mut options = Options::default();
+        options.lock_ttl = ttl;
+        options.min_commit_ts = ts;
+        options.for_update_ts = for_update_ts;
+        must_prewrite_put_impl(engine, key, value, pk, ts, for_update_ts != 0, options);
```

Comment:
`for_update_ts != 0` is a necessary condition, not a sufficient condition for a pessimistic lock. But it doesn't matter for test.